js/windowMgr.js

```diff
@@ -319,7 +316,6 @@ function doCreateMainWindow(initialUrl, initialBounds) {
         }
     });
 
```

Comment:
can you document why we've removed the contextMenu and what is the advantage / consequence of the same?